src/ScoutingMain.elm

```diff
@@ -0,0 +1,109 @@
+module ScoutingMain exposing (Model, Msg)
+
+import Browser
+import Debug
+import Html exposing (button, div, input, label, text)
+import Html.Attributes as Attributes exposing (placeholder, style, type_, value)
+import Html.Events as Events exposing (onClick, onInput)
+import Http
+import Json.Decode
+import String
+
+
+main : Program () Model Msg
+main =
+    Browser.element
+        { init = always ( init, Cmd.none )
+        , view = view
+        , update = \msg model -> ( update msg model, Cmd.none )
+        , subscriptions = always subscriptions
+        }
+
+
+type Pages
+    = P1
+    | P2
+
+
+type Msg
+    = ScouterInput String
+    | TeamInput String
+    | MatchInput String
+    | DriverStationPositionInput String
+    | Move
+
+
+type alias Model =
+    { scouterName : String
+    , teamNum : Int
+    , matchNum : Int
+    , driverStationPosition : String
+    , pages : Pages
+    }
+
+
+init : Model
+init =
+    { scouterName = ""
+    , teamNum = 0
+    , matchNum = 0
+    , driverStationPosition = ""
+    , pages = P1
+    }
+
+
+checkbox : String -> (String -> Msg) -> String -> Html.Html Msg
+checkbox modelValue nextButton name =
+    div []
+        [ input [ placeholder name, onInput nextButton, value modelValue ] [] ]
```

Comment:
You should use `type_ "checkbox"` as demonstrated here: https://github.com/dwyl/learn-elm/blob/master/examples/checkboxes.elm#L69